package.json

```diff
@@ -24,7 +24,8 @@
     "gtoken": "^2.1.0",
     "jws": "^3.1.4",
     "lodash.isstring": "^4.0.1",
-    "lru-cache": "^4.1.1"
+    "lru-cache": "^4.1.1",
+    "retry-axios": "^0.3.0"
```

Comment:
Is this the same thing we use in our client libraries to do retries? If not, could we use that? If not, why not?

Reply:
Do you think it's an okay state to be in that our libraries use multiple http libraries and multiple retry libraries?

Reply:
It's not ideal, and should probably change.  I'd love to have a plan to migrate us 100% of axios, but it's beyond the scope of what I can accomplish in this library.  Happy to offer guidance and input though :)

Reply:
The alternatives are:
* Continue using a client library that has problems.
* Rewrite all code to switch atomically to a different library.

There may be *some* downsides of multiple client libraries; but those are unspecified in this thread and assumed to be terrible. IMHO they aren't terrible.

Reply:
It was an honest question - I am ramping up on all things in all languages and trying to figure out what matters and what doesn't.

For context, in Python we were faced with a *very* similar issue - we used httplib2 throughout and it was horrible. We needed to rapidly move to requests, starting by allowing the auth library to use [*multiple* transports](http://google-auth.readthedocs.io/en/latest/reference/google.auth.transport.html). We then updated our libraries one by one to use requests instead of httplib2. There was a little overlap where two different libraries would use different transports, but within a single library one transport was used. It took ~month to get everything moved over.

This was a much bigger concern in Python than my current understanding leads me to believe it is in node.